---
title: "Stats key resources"
author: "Nicholas W Daudt"
date: "02/05/2023"
output:
  html_document:
    toc: true
    number_sections: true
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Before anything else -- must-reads

* **Zuur *el al.* (2010)** [A protocol for data exploration to avoid common statistical problems](https://doi.org/10.1111/j.2041-210X.2009.00001.x). *Methods in Ecology and Evolution* 1: 3-14.
* **Tredennick *et al.* (2021)** [A practical guide to selecting models for exploration, inference, and prediction in ecology](https://esajournals.onlinelibrary.wiley.com/doi/abs/10.1002/ecy.3336). *Ecology* 102: e03336.
* **Edwards & Auger-Méthé (2019)** [Some guidance on using mathematical notation in ecology](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13105). *Methods in Ecology and Evolution* 10: 92-99.

# Univariate

## ANOVA

1. Gelman (2005) [Analysis of variance — why it is more important than ever](https://projecteuclid.org/journals/annals-of-statistics/volume-33/issue-1/Analysis-of-variancewhy-it-is-more-important-than-ever/10.1214/009053604000001048.full). *The Annals of Statistics* 33: 1-53.
2. Hector *et al.* (2010) [Analysis of variance with unbalanced data: an update for ecology & evolution](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1365-2656.2009.01634.x). *Journal of Animal Ecology* 79: 308-316.

## GL(M)M/GA(M)M

* Santon *et al.* (2023) [A versatile workflow for linear modelling in R](https://doi.org/10.3389/fevo.2023.1065273). *Frontier in Ecology and Evolution* 11: 1065273.
* Duncan & Kefford (2021) [Interactions in statistical models: Three things to know](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13714). *Methods in Ecology and Evolution* 12: 2287-2297.

1. Venalbles & Dichmont (2004) [GLMs, GAMs and GLMMs: an overview of theory for applications in fisheries research](https://doi.org/10.1016/j.fishres.2004.08.011). *Fisheries Research* 70: 319-337.
2. Bolker *et al.* (2009) [Generalized linear mixed models: a practical guide for ecology and evolution](https://doi.org/10.1016/j.tree.2008.10.008). *Trends in Ecology & Evolution* 24: 127-135.
3. O'Hara (2009) [How to make models add up — a primer on GLMMs](http://www.bioone.org/doi/abs/10.5735/086.046.0205). *Annales Zoologici Fennici* 46: 124-137.
4. Schielzeth & Nakagawa (2013) [Nested by design: model fitting and interpretation in a mixed model era](http://doi.wiley.com/10.1111/j.2041-210x.2012.00251.x). *Methods in Ecology and Evolution* 4: 14-24.
5. Harrison *et al.* (2018) [A brief introduction to mixed effects modelling and multi-model inference in ecology](https://peerj.com/articles/4794). *PeerJ* 6: e4794.
6. Pedersen *et al.* (2019) [Hierarchical generalized additive models in ecology: an introduction with mgcv](https://peerj.com/articles/6876). *PeerJ* 7: e6876.

## GAMLSS

1. Rigby & Stasinopoulos (2005) [Generalized additive models for location, scale and shape](https://rss.onlinelibrary.wiley.com/doi/abs/10.1111/j.1467-9876.2005.00510.x). *Journal of the Royal Statistical Society: Series C (Applied Statistics)* 54: 507-554.

## Regression models for count data

* O'Hara & Kotze (2010) [Do not log-transform count data](http://doi.wiley.com/10.1111/j.2041-210X.2010.00021.x). *Methods in Ecology and Evolution* 1: 118-122.
* Warton *et al.* (2016) [Three points to consider when choosing a LM or GLM test for count data](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.12552). *Methods in Ecology and Evolution* 7: 882-890.

1. Zeiles *et al.* (2008) [Regression Models for Count Data in R](http://www.jstatsoft.org/v27/i08/). *Journal of Statistical Software* 27: 1-25.

## Species distribution modelling (*lato sensu*)

<!-- https://onlinelibrary.wiley.com/toc/13652699/2012/39/12 -->

* Soberón & Nakamura (2009) [Niches and distributional areas: concepts, methods, and assumptions](https://pnas.org/doi/full/10.1073/pnas.0901637106). *Proceedings of the National Academy of Sciences* 106: 19644-19650.
* Peterson & Soberón (2012) [Species istribution Modeling and Ecological Niche Modeling: getting the concepts right](http://dx.doi.org/10.4322/natcon.2012.019). *Natureza & Conservação* 10: 102-107.
* Elith & Leathwick (2009) [Species distribution models: ecological explanation and prediction across space and time](http://www.annualreviews.org/doi/10.1146/annurev.ecolsys.110308.120159). *Annual Review of Ecology, Evolution, and Systematics* 40: 677-697.

1. Elith *et al.* (2011) [A statistical explanation of MaxEnt for ecologists](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1472-4642.2010.00725.x). *Diversity and Distributions* 17: 43-57.
2. Elith & Graham (2009) [Do they? How do they? WHY do they differ? On finding reasons for differing performances of species distribution models](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1600-0587.2008.05505.x). *Ecography* 32: 66-77.
3. Warton & Shepherd (2010) [Poisson point process models solve the "pseudo-absence problem" for presence-only data in ecology](https://www.jstor.org/stable/29765559). *The Annals of Applied Statistics* 4: 1383-1402.
4. Renner & Warton (2013) [Equivalence of MAXENT and Poisson Point Process models for species distribution modeling in ecology](https://doi.org/10.1111/j.1541-0420.2012.01824.x). *Biometrics* 69: 274-281.

## Machine Learning

* Christin *et al.* (2019) [Applications for deep learning in ecology](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13256). *Methods in Ecology and Evolution* 10: 1632-1644.
* Borowiec *et al.* (2022) [Deep learning as a tool for ecology and evolution](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13901). *Methods in Ecology and Evolution* 13: 1640-1660.
* Pichler & Hartig (2023) [Machine learning and deep learning—A review for ecologists](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14061). *Methods in Ecology and Evolution* 14: 994-1016.

### Random forest

1. Cutler *et al.* (2007) [Random forests for classification in ecology](https://doi.org/10.1890/07-0539.1). *Ecology* 88: 2783-2792.
2. Strobl *et al.* (2007) [Bias in random forest variable importance measures: Illustrations, sources and a solution](https://doi.org/10.1186/1471-2105-8-25). *BMC Bioinformatics* 8: 25. 
3. Genuer *et al.* (2010) [Variable selection using random forests](https://doi.org/10.1016/j.patrec.2010.03.014). *Pattern Recognition Letters* 31: 2225-2236.

### Boosted regression trees

1. Elith *et al.* (2008) [A working guide to boosted regression trees](https://doi.org/10.1111/j.1365-2656.2008.01390.x). *Journal of Animal Ecology* 77: 802-813.
2. Mayr & Hofner (2018) [Boosting for statistical modelling - A non-technical introduction](http://journals.sagepub.com/doi/10.1177/1471082X17748086). *Statistical Modelling* 18: 365-384.

## Quantile regression

1. Cade & Noon (2003) A gentle introduction to quantile regression for ecologists. *Frontiers in Ecology and the Environment* 1: 412-420. <https://doi.org/10.1890/1540-9295(2003)001[0412:AGITQR]2.0.CO;2>

## Casual inference

* Grace & Irvine (2020) [Scientist’s guide to developing explanatory statistical models using causal analysis principles](https://onlinelibrary.wiley.com/doi/abs/10.1002/ecy.2962). *Ecology* 101: e02962.
* Arif & MacNeil (2022) [Predictive models aren't for causal inference](https://onlinelibrary.wiley.com/doi/abs/10.1111/ele.14033). *Ecology Letters* 25: 1741-1745.

1. Laubach *et al.* (2021) [A biologist's guide to model selection and causal inference](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2020.2815). *Proceedings of the Royal Society B: Biological Sciences* 288: 20202815.
2. Arif & MacNeil (2022) [Utilizing causal diagrams across quasi-experimental approaches](https://onlinelibrary.wiley.com/doi/abs/10.1002/ecs2.4009). *Ecosphere* 13: e4009.
3. Gao *et al.* (2022) [Causal inference in spatial statistics](https://doi.org/10.1016/j.spasta.2022.100621). *Spatial Statistics* 50: 100621.
4. Arif & MacNeil (2023) [Applying the structural causal model framework for observational causal inference in ecology](https://doi.org/10.1002/ecm.1554). *Ecological Monographs* 93: e1554.

### Structural Equation Modelling

1. Grace & Bollen (2005) [Interpreting the results from Multiple Regression and Structural Equation Models](https://esajournals.onlinelibrary.wiley.com/doi/abs/10.1890/0012-9623%282005%2986%5B283%3AITRFMR%5D2.0.CO%3B2). *The Bulletin of the Ecological Society of America* 86: 283-295.
2. Grace *et al.* (2010) [On the specification of structural equation models for ecological systems](https://esajournals.onlinelibrary.wiley.com/doi/abs/10.1890/09-0464.1). *Ecological Monographs* 80: 67-87.
3. Grace *et al.* (2012) [Guidelines for a graph-theoretic implementation of structural equation modeling](https://esajournals.onlinelibrary.wiley.com/doi/abs/10.1890/ES12-00048.1). *Ecosphere* 3: 1-44.
4. Fan *et al.* (2016) [Applications of structural equation modeling (SEM) in ecological studies: an updated review](https://doi.org/10.1186/s13717-016-0063-3). *Ecological Processes* 5: 19.

# Occupancy models

* Devarajan *et al.* (2020) [Multi-species occupancy models: review, roadmap, and recommendations](https://doi.org/10.1111/ecog.04957). *Ecography* 43: 1612-1624.

Doser *et al.* (2022) [spOccupancy: An R package for single-species, multi-species, and integrated spatial occupancy models](https://doi.org/10.1111/2041-210X.13897). *Methods in Ecology and Evolution* 13: 1670-1678.

# Multivariate

* Ferrier & Guisan (2006) [Spatial modelling of biodiversity at the community level](https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/j.1365-2664.2006.01149.x). *Journal of Applied Ecology* 43: 393-404.

## Model-based: WHYs

**Ecology**

* Ovaskainen & Soininen (2011) [Making more out of sparse data: hierarchical modeling of species communities](https://esajournals.onlinelibrary.wiley.com/doi/abs/10.1890/10-1251.1). *Ecology* 92: 289-295.
* Warton *et al.* (2015) [So many variables: joint modeling in community ecology](https://doi.org/10.1016/j.tree.2015.09.007). *Trends in Ecology & Evolution* 30: 766-779.
* Warton *et al.* (2015) [Model-based thinking for community ecology](https://doi.org/10.1007/s11258-014-0366-3). *Plant Ecology* 216: 669-682.
* Ovaskainen *et al.* (2017) [How to make more out of community data? A conceptual framework and its implementation as models and software](). *Ecology Letters* 20: 561-576.

**Statistics**

* Warton *et al.* (2012) [Distance-based multivariate analyses confound location and dispersion effects](http://doi.wiley.com/10.1111/j.2041-210X.2011.00127.x). *Methods in Ecology and Evolution* 3: 89-101.
* Warton & Hui (2017) [The central role of mean-variance relationships in the analysis of multivariate abundance data: a response to Roberts (2017)](https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.12843). *Methods in Ecology and Evolution* 8: 1408-1414.

## Model-based: Ordination

1. Hui (2015) [Model‐based approaches to unconstrained ordination](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.12236). *Methods in Ecology and Evolution* 6: 6: 399-411.
2. van der Veen *et al.* (2021) [Model-based ordination for species with unequal niche widths](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13595). *Methods in Ecology and Evolution* 12: 1288-1300.
3. Popovic *et al.* (2022) [Fast model-based ordination with copulas](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13733). *Methods in Ecology and Evolution* 13: 194-202.
4. van der Veen *et al.* (2023) [Concurrent ordination: Simultaneous unconstrained and constrained latent variable modelling](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.14035). *Methods in Ecology and Evolution* 14: 683-695.

## Model-based: Joint 'SDMs'

* Poggiato *et al.* (2021) [On the interpretations of joint modeling in community ecology](https://doi.org/10.1016/j.tree.2021.01.002). *Trends in Ecology and Evolution* 36: 391-401.

## Model-based: Mixture-models (+ SAM, RCP)

* Hui *et al.* (2013) [To mix or not to mix: comparing the predictive performance of mixture models vs. separate species distribution models](https://esajournals.onlinelibrary.wiley.com/doi/abs/10.1890/12-1322.1). *Ecology* 94: 1913-1919.
* Dunstan *et al.* (2013) [Finite mixture of regression modeling for high-dimensional count and biomass data in ecology](https://doi.org/10.1007/s13253-013-0146-x). *Journal of Agricultural, Biological, and Environmental Statistics* 18: 357-375.

**SAM:**

1. Dunstan *et al.* (2011) [Model based grouping of species across environmental gradients](https://www.sciencedirect.com/science/article/pii/S0304380010006393). *Ecological Modelling* 222: 955-963.

**RCP:**

1. Foster *et al.* (2013) [Modelling biological regions from multi-species and environmental data](https://onlinelibrary.wiley.com/doi/10.1002/env.2245). *Environmetrics* 24: 489-499.
2. Foster *et al.* (2017) [Ecological grouping of survey sites when sampling artefacts are present](https://rss.onlinelibrary.wiley.com/doi/abs/10.1111/rssc.12211). *Journal of the Royal Statistical Society: Series C (Applied Statistics)* 66: 1031-1047.

## Model-based: Generalized Dissimilarity Model

1. Ferrier *et al.* (2007) [Using generalized dissimilarity modelling to analyse and predict patterns of beta diversity in regional biodiversity assessment](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1472-4642.2007.00341.x). *Diversity and Distributions* 13: 252-264.
2. Latombe *et al.* (2017) [Multi-site generalised dissimilarity modelling: using zeta diversity to differentiate drivers of turnover in rare and widespread species](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.12756). *Methods in Ecology and Evolution* 8: 431-442.
3. Woolley *et al.* (2017) [Characterising uncertainty in generalised dissimilarity models](https://onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.12710). *Methods in Ecology and Evolution* 8: 985-995.
4. Mokany *et al.* (2022) [A working guide to harnessing generalized dissimilarity modelling for biodiversity analysis and conservation assessment](https://onlinelibrary.wiley.com/doi/abs/10.1111/geb.13459). *Global Ecology and Biogeography* 31: 802-821.

## Algorithm-based: general

1. Legendre & Gallagher (2001) [Ecologically meaningful transformations for ordination of species data](https://doi.org/10.1007/s004420100716). *Oecologia* 129: 271-280.
2. Peres-Neto & Jackson (2001) [The importance of scaling of multivariate analysis in ecological studies](https://www.tandfonline.com/doi/full/10.1080/11956860.2001.11682683). *Écoscience* 8: 522-526.
3. Ramette (2007) [Multivariate analyses in microbial ecology](https://doi.org/10.1111/j.1574-6941.2007.00375.x). *FEMS Microbiology Ecology* 62: 142-160.

# Spatial data / stats

* Legendre & Fortin (1989) [Spatial pattern and ecological analysis](https://doi.org/10.1007/BF00048036). *Vegetatio* 80: 107-138
* Wiens (1989) [Spatial scaling in ecology](https://www.jstor.org/stable/2389612). *Functional Ecology* 3: 385-397.
* Levin (1992) [The problem of pattern and scale in ecology](https://onlinelibrary.wiley.com/doi/abs/10.2307/1941447). *Ecology* 73: 1943-1967.
* Horne & Schneider (1995) [Spatial variance in ecology](https://www.jstor.org/stable/3545670). *Oikos* 74: 18-26.
* Brown (1995) [Spatial variation in abundance](http://doi.wiley.com/10.2307/1941678). *Ecology* 76: 2028-2043.

1. Dale *et al.* (2002) [Conceptual and mathematical relationships among methods for spatial analysis](https://onlinelibrary.wiley.com/doi/abs/10.1034/j.1600-0587.2002.250506.x). *Ecography* 25: 558-577.
2. Perry *et al.* (2002) [Illustrations and guidelines for selecting statistical methods for quantifying spatial pattern in ecological data](https://onlinelibrary.wiley.com/doi/abs/10.1034/j.1600-0587.2002.250507.x). *Ecography* 25: 578-600.

## Spatial autocorrelation

1. Sokal & Oden (1978) [Spatial autocorrelation in biology: 1. Methodology](https://academic.oup.com/biolinnean/article-lookup/doi/10.1111/j.1095-8312.1978.tb00013.x). *Biological Journal of the Linnean Society* 10: 199-228.
2. Legendre (1993) [Spatial autocorrelation: trouble or new paradigm?](http://doi.wiley.com/10.2307/1939924). *Ecology* 74: 1659-1673.
3. **Dormann *et al.* (2007)** [Methods to account for spatial autocorrelation in the analysis of species distributional data: a review](https://onlinelibrary.wiley.com/doi/abs/10.1111/j.2007.0906-7590.05171.x). *Ecography* 30: 609-628.

# Bayesian

* Ellison (2004) [Bayesian inference in ecology](http://doi.wiley.com/10.1111/j.1461-0248.2004.00603.x). *Ecology Letters* 7: 509-520.
* Hooten & Hobbs (2015) [A guide to Bayesian model selection for ecologists](http://doi.wiley.com/10.1890/14-0661.1). *Ecological Monographs* 85: 3-28.

# Miscellaneous

## General resources

1. [BMJ - Statistics Notes in the British Medical Journal](https://www-users.york.ac.uk/~mb55/pubs/pbstnote.htm)
2. [Nature - "Statistics for Biologists": Points of Significance](https://www.nature.com/collections/qghhqm/pointsofsignificance)

## P-values

1. Goodman (2008) [A dirty dozen: twelve P-value misconceptions](https://doi.org/10.1053/j.seminhematol.2008.04.003). *Seminars in Hematology* 45: 135-140.
2. Muff *et al.* (2021) [Rewriting results sections in the language of evidence](https://doi.org/10.1016/j.tree.2021.10.009). *Trends in Ecology & Evolution* 37: 203-210.

## Zeroes

1. Martin *et al.* (2005) [Zero tolerance ecology: improving ecological inference by modelling the source of zero observations](https://doi.org/10.1111/j.1461-0248.2005.00826.x). *Ecology Letters* 8: 1235-1246.
2. Warton (2005) [Many zeros does not mean zero inflation: comparing the goodness-of-fit of parametric models to multivariate abundance data](https://onlinelibrary.wiley.com/doi/abs/10.1002/env.702). *Environmetrics* 16: 275-289.
3. Blasco-Moreno *et al.* (2019) [What does a zero mean? Understanding false, random and structural zeros in ecology](https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/2041-210X.13185). *Methods in Ecology and Evolution* 10: 949-959.
